jobs/Backend/Task/ExchangeRateProvider/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ExchangeRateProvider
+{
+    class Program
+    {
+        private const string ratesUrl = @"https://www.cnb.cz/en/financial-markets/foreign-exchange-market/central-bank-exchange-rate-fixing/central-bank-exchange-rate-fixing/daily.txt";
```

Comment:
this url provides just preselected exchange rates. There's also another url provided by cnb where we can find the rest of exchange rates. 
For the purposes of this task I have decided to use this url.